packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -137,9 +137,13 @@ class PreferencesController extends DisposableController
   }
 }
 
+enum InspectorDetailsViewOptions { layoutExplorer, widgetDetailsView }
```

Comment:
supernit: InspectorDetailsViewType to match singular and "Type" word pattern we use elsewhere in DevTools